web/vtctld2/src/app/dashboard/shard.component.ts

```diff
@@ -244,8 +291,9 @@ export class ShardComponent implements OnInit, OnDestroy {
   }
 
   openDeleteTabletDialog() {
-    this.dialogSettings = new DialogSettings('Delete', `Delete ${this.selectedTablet.label}`,
-                                             `Are you sure you want to delete ${this.selectedTablet.label}?`,
+    this.dialogSettings = new DialogSettings('Delete Tablet', `Delete ${this.selectedTablet.label}`,
+                                             `Are you sure you want to delete ${this.selectedTablet.label}?` +
+                                             ` It will be removed from topology (but vttablet and MySQL won't be touched).`,
```

Comment:
nit: *the* topology

We're talking about the user's topology and not in general.